### PR TITLE
MAUI-5256 : Prepare styles UG for .NET MAUI Popup

### PR DESCRIPTION
**Pull request by RiyashameedM**
Prepared styles UG for .NET MAUI Popup

### CODE REVIEW

File: MAUI/Popup/styles.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline MAUI/Popup/styles.md:2]: Layout: Post

File: MAUI/Popup/styles.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+title: Styles in .NET MAUI Popup control | Syncfusion
```
Comment (Chrispineagunja) [inline MAUI/Popup/styles.md:3]: Title: Styles in .NET MAUI Popup control | Syncfusion

File: MAUI/Popup/styles.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+description: Learn here all about Styles support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
```
Comment (Chrispineagunja) [inline MAUI/Popup/styles.md:4]: Description: Learn about Styles support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements, and more.

File: MAUI/Popup/styles.md [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+platform: MAUI
```
Comment (Chrispineagunja) [inline MAUI/Popup/styles.md:5]: Platform: MAUI

File: MAUI/Popup/styles.md [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+control: SfPopup
```
Comment (Chrispineagunja) [inline MAUI/Popup/styles.md:6]: Control: SfPopup

File: MAUI/Popup/styles.md [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline MAUI/Popup/styles.md:7]: Documentation: UG

File: MAUI/Popup/styles.md [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+--- 
+
+# Styles in .NET MAUI Popup (SfPopup)
+
+The SfPopup applies style to all of its elements by using the `SfPopup.PopupStyle` property.
+
+## Styling popup header
+
+The SfPopup allows customizing the header appearance using the below properties.
```
Comment (Chrispineagunja) [inline MAUI/Popup/styles.md:16]: The SfPopup allows customizing the header appearance using the following properties.

File: MAUI/Popup/styles.md [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup with overlay customization](/Images/styles/maui-popup-with-overlay-customization.png)
+
+### Blurred background
+
+The SfPopup allows to blur the background using `SfPopup.OverlayMode` and `SfPopup.PopupStyle.BlurIntensity` properties, respectively.
```
Comment (Chrispineagunja) [inline MAUI/Popup/styles.md:238]: The SfPopup allows blurring of the background using the `SfPopup.OverlayMode` and `SfPopup.PopupStyle.BlurIntensity` properties respectively.

File: MAUI/Popup/styles.md (hunk 3 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:4]: modified

File: MAUI/Popup/styles.md (hunk 8 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:238]: modified

File: MAUI/Popup/styles.md (hunk 7 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:16]: modified

File: MAUI/Popup/styles.md (hunk 1 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:2]: not changed

File: MAUI/Popup/styles.md (hunk 4 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:5]: not changed

File: MAUI/Popup/styles.md (hunk 5 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:6]: not changed

File: MAUI/Popup/styles.md (hunk 6 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:7]: not changed

**Pull request closed without merging**